### ISSUE TITLE
It is downloading thumbnail instead of the HD Dp.

### ISSUE THREAD

**Issue by aumirza** [sdushantha/instadp#5]

**Comment by sdushantha** [comment]
@aumirza Yes, I am aware of that. This is because Instagram did something with their HTML and I cant seem to find the HD image. If you find where the HD image is located, please let me know :)
